CRM/Contribute/Form/AbstractEditPayment.php

```diff
@@ -289,7 +289,20 @@ public function buildValuesAndAssignOnline_Note_Type($id, &$values) {
       $this->_noteID = $daoNote->id;
       $values['note'] = $daoNote->note;
     }
-    $this->_contributionType = $values['financial_type_id'];
+    $this->getFinancialTypeId();
+  }
+
+  /**
+   * Get the financial Type ID for the contribution either from the submitted values or from the contribution values if possible
+   */
+  protected function getFinancialTypeId() {
```

Comment:
@seamuslee001 I think this function should return the financialTypeID (I also think we are more on the caps id than the camel one on these things in general)

Reply:
@eileenmcnaughton I have just tried this but have failed to make it work will try again tomorrow but yeh struggling a bit